src/utils/create-networked-entity.ts

```diff
@@ -25,12 +25,12 @@ export function renderAsNetworkedEntity(
   nid: NetworkID,
   creator: ClientID
 ) {
-  const eid = renderAsEntity(world, prefabs.get(prefabName)!.template(initialData));
+  const eid = renderAsEntity(world, prefabs.get(prefabName)!.template(initialData as any));
```

Comment:
I think we looked into this already and the answer was no. But can't this be an `InitialData` instead of `any`?